OpenDreamServer/Dream/Objects/DreamObject.cs

```diff
@@ -76,6 +76,14 @@ class DreamObject {
             }
         }
 
+        public List<DreamValue> GetVariables() {
```

Comment:
The name of this seems like it could be misleading, maybe `GetVariableNames()`?

Reply:
I was just going off the other method, will change